modules/consentManagement.js

```diff
@@ -232,6 +233,10 @@ export function requestBidsHook(fn, reqBidsConfigObj) {
  * @param {object} hookConfig contains module related variables (see comment in requestBidsHook function)
  */
 function processCmpData(consentObject, hookConfig) {
+  // always send gdprApplies: true to adapters when gdprScope is true
+  if (gdprScope === true && consentObject && consentObject.getConsentData) {
```

Comment:
I may be reading this wrong, but not sure this works at a high level -- the goal is to pass defaultGdprScope to adapters even when the CMP doesn't respond. So please confirm that `processCmpData` is the right place to put this logic. If there's a scenario where `processCmpData` isn't called, then we need another place that creates consentObject.getConsentData. gdprApplies

Reply:
we probably want to do this in the `storeConsentData` function instead of `processCmpData`.  The store is called even if there was an error.

At a pinch, we could also try the `gdprDataHandler` object in `adapterManager.js`.